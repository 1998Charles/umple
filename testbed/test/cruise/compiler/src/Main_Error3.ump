namespace compiler;

class MainError3
{
    public static void main(String [] argv){
        int a = 1;
        b = a;
    }
}
