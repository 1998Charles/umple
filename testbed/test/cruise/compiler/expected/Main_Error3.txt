Main_Error3.ump:7: error: cannot find symbol
        b = a;
        ^
  symbol:   variable b
  location: class MainError3
1 error
